docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -215,12 +215,66 @@ On Nextcloud inside the _OpenProject Integration_ App, when adding the OpenProje
+- Set up the project folders
+
+> Note: You need to disable the _OpenProject Integration_ App because user/group `OpenProject` is protected by the app, and it won't allow you to delete the user/group named "OpenProject". Disabling and enabling the app is safe. No app data will be deleted.
+
+If you do care about the `OpenProject` user/group/folder data then the conditions that bring the project folder setup to error state, and its possible solutions are as listed below: 
+
+- User `OpenProject` doesn't exist. Please check if the user exists, if not create a user named `OpenProject` with username and display name `OpenProject`, some secure random password, email is not necessary.
+- Group `OpenProject` doesn't exist. Please check if the group exists, if not create a group named `OpenProject`.
+- User `OpenProject` is not a member of group `OpenProject`. Please check the user is member of the group, if not add the user `OpenProject` to the group `OpenProject`.
+- User `OpenProject` is not group admin of group `OpenProject`. Please check the user is admin of the group, if not make user `OpenProject` the group admin of group `OpenProject`
+- Group folder `OpenProject` doesn't exist. If you don't have the _Group folders_ App installed, please install and enable it. Inside the _Group folders_ App (*Administration settings → Administration → Group folders*) make a group folder named `OpenProject`. Add the group `OpenProject` to the group folder with all the permissions i.e. Write, Share, Delete. Add user `OpenProject` to advance permissions list.
+- Group folder `OpenProject` is not assigned to group "OpenProject". Inside the _Group folders_ App (*Administration settings → Administration → Group folders*) check if the "OpenProject" group folder has group "OpenProject" assigned to it, if not add the group "OpenProject" with all the permissions i.e. Write, Share, Delete.
+- Group `OpenProject` doesn't have all the permissions for group folder `OpenProject`. Inside the _Group folders_ App (*Administration settings → Administration → Group folders*) check if the group "OpenProject" has all the permissions for group folder "OpenProject", if not give group "OpenProject" all the permissions i.e. Write, Share, Delete.
+- User `OpenProject` doesn't have advanced permissions for group folder `OpenProject`.  Inside the _Group folders_ App (*Administration settings → Administration → Group folders*) check if the user "OpenProject" has advanced permissions for group folder "OpenProject", if not add user "OpenProject" to advanced permissions list.
+
+> Note: The name `OpenProject` is case-sensitive, so should be in exact that format.
+
+#### While trying to delete or disable user/group "OpenProject"
+
+If you face an error while trying to delete or disable user/group "OpenProject" then that's because user/group is protected by _OpenProject Integration_ App. If you really need to delete the user or group follow following steps:
```

Comment:
```suggestion
If you face an error while trying to delete or disable user/group "OpenProject" then that's because user/group is protected by the _OpenProject Integration_ App. If you really need to delete the user or group follow these steps:
```